src/main/java/org/symphonyoss/symphony/messageml/elements/Form.java

```diff
@@ -19,6 +19,11 @@ public Form(Element parent) {
     super(parent, MESSAGEML_TAG);
   }
 
+  @Override
+  protected Boolean hasIdAttribute() {
```

Comment:
You can keep this as public, or if you leave as protected this should be below the public ones.